{"version":1,"ops":[{"type":1,"author":{"id":"56b030459ff5c3e63b58d71e1bed10897e6a7686"},"timestamp":1257122539,"metadata":{"github-id":"MDU6SXNzdWUyODQ4NjM3","github-url":"https://github.com/parrot/parrot/issues/430","origin":"github"},"title":"Lexicals and :outer not Thawed Properly from PBC","message":"Several tests in t/op/lexicals.t fail when run as PBC (or with parrot -r):\n\nt/op/lexicals (Wstat: 1024 Tests: 47 Failed: 4)\nFailed tests: 38, 40-41, 43\n\n$ ./parrot t/op/lexicals_38.pbc\n20\nBug: Context 811aee8 :outer points back to itself.\ncurrent instr.: '\u0026g' pc 21 \n(/home/chromatic/dev/parrot/t/op/lexicals_38.pir:17)\ncalled from Sub '\u0026main' pc 56 \n(/home/chromatic/dev/parrot/t/op/lexicals_38.pir:26)\n\n$ ./parrot t/op/lexicals_40.pbc\nNull PMC access in get_string()\ncurrent instr.: '\u0026g' pc 9 (/home/chromatic/dev/parrot/t/op/lexicals_40.pir:14)\ncalled from Sub '\u0026main' pc 33 \n(/home/chromatic/dev/parrot/t/op/lexicals_40.pir:20)\n\n$ ./parrot t/op/lexicals_41.pbc\nNull PMC access in get_string()\ncurrent instr.: '\u0026g' pc 9 (/home/chromatic/dev/parrot/t/op/lexicals_41.pir:15)\ncalled from Sub '\u0026main' pc 43 \n(/home/chromatic/dev/parrot/t/op/lexicals_41.pir:22)\n\n$ ./parrot t/op/lexicals_43.pbc\n(hangs; interrupted)\n\nI suspect these are all the same problem, and are related to RT #60650.\n\n-- c\n\nOriginally http://trac.parrot.org/parrot/ticket/1171","files":null}]}